modules/openapi-generator/src/main/resources/JavaJaxRS/spec/beanValidation.mustache

```diff
@@ -1,4 +1 @@
-{{#required}}
-  @NotNull
-{{/required}}
-{{>beanValidationCore}}
\ No newline at end of file
+{{#required}}{{^isReadOnly}}@NotNull {{/isReadOnly}}{{/required}}{{#isContainer}}{{^isPrimitiveType}}{{^isEnum}}@Valid {{/isEnum}}{{/isPrimitiveType}}{{/isContainer}}{{^isContainer}}{{^isPrimitiveType}}@Valid {{/isPrimitiveType}}{{/isContainer}}{{>beanValidationCore}}
```

Comment:
The content is coming from here:
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/JavaSpring/beanValidation.mustache